dashboard/app/email_test.go

```diff
@@ -147,11 +148,15 @@ For more options, visit https://groups.google.com/d/optout.
 			"always@cc.me",
 			"bugs2@syzkaller.com",
 			"bugs@syzkaller.com", // This is from incomingEmail.
+			"crashCC@foo.com",
 			"default@sender.com", // This is from incomingEmail.
 			"foo@bar.com",
 			config.Namespaces["test2"].Reporting[0].Config.(*EmailConfig).Email,
 		}
+
+		cc := []string{}
 		c.expectEQ(msg.To, to)
+		c.expectEQ(msg.Cc, cc)
```

Comment:
Would nil work here? Creating a new variable and consuming a line of code for an empty slice looks excessive.